code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -509,11 +509,45 @@
 	taste_description = "bitterness"
 	reagent_state = LIQUID
 	color = "#C1C1C1"
-	metabolism = REM * 0.05
```

Comment:
This is being sent to polaris, so don't need to mark as edit unless they don't do it.
I noticed this 20x increase in metabolism rate also, I am not sure its correct or not.

Reply:
Given that it's actually able to be more powerful now (previously higher doses didn't do anything), and there's an even more powerful antibiotic, plus that pills now work so you can just give large doses/pillbottles to patients (which is basically how it'd go in RL), I'm not too worried about the metabolism rate. I can make it *0.5 if you want it to last a smidge longer.

Reply:
Gotta remember that this is also the first line of defense against viruses, too. Polaris still has those, iirc. (I think it'd also be cool if we had a virus encounter on a future exploration map.)


Also even at OD rates prior, it was still slow to work, and keeping it at OD levels with fast metabolism would be pretty hard without vastly overdosing them. I feel like keeping it at the rate it was at would be more prudent overall.

Reply:
I didn't change anything about it's effect on viruses. This only changes it's effect for infections.

Reply:
No, but what I mean is if it metabolizes quickly, it will require more and larger doses to stave off virus effects. This would grow exponentially for each patient. There's almost never enough to go around as is on Polaris; nerfing it that hard will only make it even harder to deal with viruses.

Reply:
Lowered the metab rate. I still don't feel five-hundredths the normal rate is appropriate though. It may have been when syringes were the only way, but given that it doesn't require phoron in it's creation and you can just make pillbottles to hand out to people now, it's fine.

The idea that someone should be entirely fixed before leaving medical is, in my opinion, sort of nonsense. If you've given them a proscription for spaceacillin and tell them to pop one pill every 10 minutes, that's far better than "you have to stand here".

Reply:
Seems good then! 

Also yes I seriously wish that I could refactor medical entirely and make injury more punishing and realistic than it is.